### PR TITLE
feat: Style suggestion popup (WEBAPP-5461)

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: app/script/components/mentionSuggestions.js [hunk 1]
```diff
@@ -111,12 +111,18 @@ z.components.MentionSuggestions = class MentionSuggestions {
...
   template: `
   <!-- ko if: isVisible() -->
-    <div class="conversation-input-bar-mention-suggestion" data-uie-name="list-mention-suggestions" data-bind="style: positions()">
-      <ul class="mention-suggestion-list" data-bind="foreach: {data: suggestions, as: 'suggestion'}">
-        <li data-bind="style: {color: suggestion === $parent.selectedSuggestion() ? 'green' : ''}">
-          <span data-bind="text: suggestion.username()"></span>
-        </li>
-      </ul>
+    <div class="
+    " data-uie-name="list-mention-suggestions" data-bind="style: positions()">
```
Comment (gregor) [inline app/script/components/mentionSuggestions.js]: This is weird...

File: app/style/components/mention-suggestions.less [hunk 2]
```diff
@@ -17,7 +17,62 @@
...
+
+      svg {
+        width: 12px;
+        height: 12px;
+        path {
+          fill: fade(@graphite-dark, 40%);
+        }
+      }
+    }
+  }
```
Comment (gregor) [inline app/style/components/mention-suggestions.less:77]: I am missing the full width definition with right margin of the complete list that is mentioned in the spec.

Comment (AndyLnd) [reply app/style/components/mention-suggestions.less:77]: We are not doing the full width anymore. And this should be reflected in the specs.

Comment (gregor) [reply app/style/components/mention-suggestions.less:77]: So the width will jump depending on which suggestions get shown in the list?

Comment (AndyLnd) [reply app/style/components/mention-suggestions.less:77]: Yes, we will see how that works out and might introduce a maximum width.

**Pull request merged**